pkg/util/typeutil/schema.go

```diff
@@ -689,6 +689,16 @@ func MergeFieldData(dst []*schemapb.FieldData, src []*schemapb.FieldData) error
 	return nil
 }
 
+// GetVectorFieldSchema get vector field schema from collection schema.
+func GetVectorFieldSchema(schema *schemapb.CollectionSchema) (*schemapb.FieldSchema, error) {
+	for _, fieldSchema := range schema.Fields {
+		if fieldSchema.DataType == schemapb.DataType_BinaryVector || fieldSchema.DataType == schemapb.DataType_FloatVector {
```

Comment:
using typeutil.IsVectorType would be better

Reply:
ok